p2p/src/network/kad/bootstrap/p2p_network_kad_bootstrap_actions.rs

```diff
@@ -45,3 +47,25 @@ impl From<P2pNetworkKadBootstrapAction> for P2pAction {
         P2pNetworkKadAction::Bootstrap(value).into()
     }
 }
+
+// impl ActionEvent for P2pNetworkKadBootstrapAction {
```

Comment:
@akoptelov why is this one commented out?

Reply:
nevermind, just saw https://github.com/openmina/openmina/pull/325#issuecomment-2037316688

Reply:
This is an example of equivalent code written manually. I'm going to remove it before merging.